internal/cmd/integrations/scaffold.go

```diff
@@ -127,6 +136,23 @@ var ScaffoldCmd = &cobra.Command{
 			return err
 		}
 
+		if len(testCasesBody) > 0 {
```

Comment:
instead of one file that has all the test cases, can we create multiple files within the testcases directory? We can have the displayName as the test case file name

Reply:
done